ci/jenkins/test-vmc.sh

```diff
@@ -459,6 +459,24 @@ function run_integration {
     fi
 }
 
+function run_multicluster_integration {
+    VM_NAME="antrea-integration-0"
+    export GOVC_URL=${GOVC_URL}
+    export GOVC_USERNAME=${GOVC_USERNAME}
+    export GOVC_PASSWORD=${GOVC_PASSWORD}
+    VM_IP=$(govc vm.ip ${VM_NAME})
+    govc snapshot.revert -vm.ip ${VM_IP} initial
+    VM_IP=$(govc vm.ip ${VM_NAME}) # wait for VM to be on
+
+    set -x
+    echo "===== Run Multi-cluster Integration tests ====="
+    # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
+    ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && cd multicluster && make test-integration"
```

Comment:
It seems the differences between `run_multicluster_integration` and `run_integration ` are just 2 lines of code. How about try to reuse most code and use 1) environment variables OR 2) parameter.

Reply:
done